modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/IconButton/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {ButtonWithIcon} from '@clayui/core';
+
+const IconButton = ({symbol, ...props}) => (
+	<ButtonWithIcon symbol={symbol} {...props} />
+);
```

Comment:
I didn't get this, what is the difference between IconButton and ButtonWithIcon if they have exactly same props?

Reply:
It's actually the same, we can use the ButtonWithIcon directly. I've just created a component in case of further customization related to it.